src/main/java/org/jala/efeeder/foodmeeting/FoodMeeting.java

```diff
@@ -12,42 +12,49 @@
  */
 @Data
 public class FoodMeeting {
-    private int id;
-    private String name;
-    private String imageLink;
-    private String status;
-    private Timestamp eventDate;
-    private Timestamp createdAt;
-
-    public FoodMeeting() {
-    }
-
-    public FoodMeeting(int id, String name,String imageLink, String status, Timestamp eventDate, Timestamp createdAt) {
-        this(id, name, imageLink, eventDate, createdAt);
-        this.status = status;
-    }
-    
-    public FoodMeeting(int id, String name,String imageLink, Timestamp eventDate, Timestamp createdAt) {
-        this.id = id;
-        this.name = name;
-        this.createdAt = createdAt;
-        this.eventDate = eventDate;
-        this.imageLink = imageLink;
-    }
-
-    public int getWidth()
-    {
-        int width = 500;
-
-        LocalDate today = new LocalDate();
-        LocalDate endEventDate = new LocalDate(eventDate);
-
-        int days = Days.daysBetween(today, endEventDate).getDays();
-
-        width = (50 * days) < 300 ?
-                width - (50 * days) :
-                200;
-
-        return width;
-    }
+	private int id;
+	private String name;
+	private String imageLink;
+	private String status;
+	private Timestamp eventDate;
+	private Timestamp createdAt;
+
+	public static final String DEFAULT_FOOD_MEETING_STATUS = FoodMeetingStatus.Voting.name();
+
+	public FoodMeeting() {
+	}
+
+	public FoodMeeting(int id, String name,String imageLink, String status, Timestamp eventDate, Timestamp createdAt) {
+		this.id = id;
+		this.name = name;
+		this.createdAt = createdAt;
+		this.eventDate = eventDate;
+		this.imageLink = imageLink;
+		this.status = status;
+	}
+
+	public FoodMeeting(int id, String name, String imageLink, Timestamp eventDate) {
+		this.id = id;
+		this.name = name;
+		this.createdAt = new Timestamp(System.currentTimeMillis());
+		this.eventDate = eventDate;
+		this.imageLink = imageLink;
+		this.status = DEFAULT_FOOD_MEETING_STATUS;
+	}
+
+	public int getWidth()
+	{
+		int width = 500;
```

Comment:
In any case, this value will be changed if you use the 'width' variable to store this value, it seems 500 is the default value to be returned